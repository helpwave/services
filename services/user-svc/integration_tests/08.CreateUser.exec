#!/bin/bash

grpcurl --plaintext -H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{"id": "948618ea-6059-4e4d-a300-6f499f79f9e3", "email": "vincent.stamm@mail.de", "name": "vincent", "nickname": "v" }' \
	localhost:3002 \
	proto.services.user_svc.v1.UserService/CreateUser 2>&1 || echo ""

echo "already exists:"
grpcurl --plaintext -H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{"id": "948618ea-6059-4e4d-a300-6f499f79f9e3", "email": "vincent.stamm@mail.de", "name": "vincent", "nickname": "v" }' \
	localhost:3002 \
	proto.services.user_svc.v1.UserService/CreateUser 2>&1 || echo ""
