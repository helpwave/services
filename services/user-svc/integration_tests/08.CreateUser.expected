{
  "id": "948618ea-6059-4e4d-a300-6f499f79f9e3"
}
